openstack/utils/templates/_hosts.tpl

```diff
@@ -24,38 +24,70 @@
 {{ include "_resolve_secret" (tuple . true) }}
 {{- end -}}
 
-{{define "db_url" }}
```

Comment:
This function isn't being used anywhere, looks like a leftover after postgres->mariadb migration

Reply:
It is used in nova for the cells.

Reply:
In the nova chart for the cells the `db_url_mysql` function is being used, not db_url directly.
db_url in the past was used conditionally if postgresql was enabled, and it's disabled for all OpenStack services now either implicitly, or explicitly with configuration:
```
postgresql:
  enabled: false
```

Reply:
Ah, no. I am mistaken.